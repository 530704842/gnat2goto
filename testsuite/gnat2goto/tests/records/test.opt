ALL XFAIL CBMC fails with symex_assign_rec: unexpected assignment to member of `symbol'
