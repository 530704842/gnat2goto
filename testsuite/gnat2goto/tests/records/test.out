[overflow.1] arithmetic overflow on signed + in records__d.records__x__a + records__c: SUCCESS
VERIFICATION SUCCESSFUL
